package.json

```diff
@@ -59,20 +84,21 @@
     "responsive"
   ],
   "dependencies": {
-    "babel-plugin-inline-package-json": "^2.0.0",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
```

Comment:
@frederickfogerty I've removed this plugin (and the newer equivalent) in the most recent commit so that consumers of this dependency don't specifically need the version we set. I was seeing some babel weirdness when integrating this with our app and realized its cause we'd declared a version new version of this as the minimum. 

Fwiw its brought in transitively via ember-cli-babel so I reckon we're safe without it?

Reply:
Sounds good to me 👍